source/app/powertabeditor.cpp

```diff
@@ -285,9 +286,12 @@ void PowerTabEditor::openFile(QString filename)
         Document &doc = myDocumentManager->addDocument(*mySettingsManager);
         myFileFormatManager->importFile(doc.getScore(), path, *format);
         auto end = std::chrono::high_resolution_clock::now();
-        qDebug() << "File loaded in"
-                 << std::chrono::duration_cast<std::chrono::milliseconds>(end - start) .count()
-                 << "ms";
+
+	const auto elapsed_time =
+		static_cast<long long int>(
```

Comment:
Thanks!  This works, but I still need to keep the `ms` units in the format string.

Reply:
Does it work if you remove the `.count()`?
From https://fmt.dev/latest/api.html#chrono-api I thought it might automatically include the ms if the type is a `chrono` type rather than an integer